owl/EASE-ACT.owl

```diff
@@ -341,6 +369,16 @@ Deduction is often explained as starting from the &quot;general&quot; (some prop
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#Distancing -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#Distancing">
```

Comment:
*BackingOff* would be more precise, maybe?

Reply:
Ok.